WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -87,39 +87,193 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
             //Seek back to the beginning of the section
             data.Seek(initialOffset, 0);
 
-            header.UnknownValue0 = data.ReadUInt32LittleEndian();
+            header.UnknownDataSize = data.ReadUInt32LittleEndian();
             header.SecondExecutableFileEntryLength = data.ReadUInt32LittleEndian();
             header.UnknownValue2 = data.ReadUInt32LittleEndian();
             header.UnknownValue3 = data.ReadUInt32LittleEndian();
             header.UnknownValue4 = data.ReadUInt32LittleEndian();
             header.FirstExecutableFileEntryLength = data.ReadUInt32LittleEndian();
             header.MsiFileEntryLength = data.ReadUInt32LittleEndian();
-            if (headerLength == 6)
-                return header;
+            if (headerLength != 6)
+            {
+                header.UnknownValue7 = data.ReadUInt32LittleEndian();
+                header.UnknownValue8 = data.ReadUInt32LittleEndian();
+                if (headerLength != 8)
+                {
+                    header.ThirdExecutableFileEntryLength = data.ReadUInt32LittleEndian();
+                    header.UnknownValue10 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue11 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue12 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue13 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue14 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue15 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue16 = data.ReadUInt32LittleEndian();
+                    header.UnknownValue17 = data.ReadUInt32LittleEndian();
+                    if (headerLength != 17)
+                    {
+                        header.UnknownValue18 = data.ReadUInt32LittleEndian();
+                    }
+                }
+            }
+            
 
-            header.UnknownValue7 = data.ReadUInt32LittleEndian();
-            header.UnknownValue8 = data.ReadUInt32LittleEndian();
-            if (headerLength == 8)
-                return header;
+            // Parse strings
+            // TODO: Count size of string section for later size verification
```

Comment:
Do you expect to address this TODO in this PR? Or is it something for later?

Reply:
Any remaining TODOs aren't going to get done this PR. I wanted to get them done for this PR, but I've taken too long and only found more and more things I need to deal with for parsing the stupid string section.

That said, I wrote this TODO earlier on in the PR. With what I know now, it might not be actually possible to verify size of the string section with how the later 4-byte... whatever it is... that's only sometimes there, but when it is, specifies things like the number of languages or sizes of one or more links in a way that can't have size fully determined before parsing.

Reply:
Okay, just wanted to gauge how much work was left.